docs: add best practice overview to help users to choose the guidelines (#127)
harvester/docs: docs/networking/best-practice/overview.md

---
sidebar_position: 51
keywords:
  - Harvester
  - harvester
  - Rancher
  - rancher
  - Network
  - network
  - VLAN
  - vlan
Description: Harvester is built on top of Kubernetes, and uses the [CNI](https://github.com/containernetworking/cni) as the interface between network providers and Kubernetes pod networking. Naturally, we implement the Harvester network based on CNI. Moreover, the Harvester UI integrates the network configuration in order to provide a user-friendly way to configure networks for VMs.
---

# Overview

> In best practice setcion, we always recommend your machine with multiple NICs for node access and VM networks. Please refer the multiple NICs best practice if your machines have multiple NICs. Otherwise, refer single NIC best practice. 

Reviewer: ```suggestion
In the [best practice](#best-practice) section below, we always recommend to configure multiple NICs in your machine for node access and VM networks. Please refer the [multiple NICs](multiple-nics-vlan-aware-switch.md) best practice if your machines have multiple NICs. Otherwise, please refer to the [single NIC](single-nic-vlan-aware-switch.md) best practice. 
```

!!! note

>     If you setup multiple NICs into a `bond`, please also consider that is single NIC scenario unless you have multiple `bond` interfaces in Harvester node.

Reviewer: ```suggestion
    If you configure a `bond` interface with multiple NICs, please refer to the single NIC scenario unless you have multiple `bond` interfaces in Harvester node.
```
